server/pkg/provision/auth.go

```diff
@@ -43,3 +61,119 @@ func emailSettings(ctx context.Context, s store.Storer) error {
 		})
 	})
 }
+
+// addAuthSuperUsers assigns BYPASS roles to users from AUTH_PROVISION_SUPER_USER value
+// When in Production, Corteza should stop and report an error.
+func addAuthSuperUsers(ctx context.Context, log *zap.Logger, s store.Storer, envOpt options.EnvironmentOpt) (err error) {
+	var (
+		val, has = os.LookupEnv("AUTH_PROVISION_SUPER_USER")
+	)
+
+	if envOpt.IsProduction() {
+		panic(errors.New(0, "Cannot provision super users in Production enviroment!"))
+	}
+
+	if has {
+		users := strings.Split(val, ";")
+
+		for _, user := range users {
+			// skip existing email
+			_, err = s.LookupUserByEmail(ctx, user)
+			if err != store.ErrNotFound {
+				log.Warn(fmt.Sprintf("Email address already %s exists", user))
+				continue
+			}
+
+			u := newUser(user)
+
+			createUserHandle(u)
+
+			// skip existing handle
+			_, err = s.LookupUserByHandle(ctx, u.Handle)
+			if err != store.ErrNotFound {
+				log.Warn(fmt.Sprintf("Handle %s already exists", u.Handle))
+				continue
+			}
+
+			if err = store.CreateUser(ctx, s, u); err != nil {
+				return err
+			}
+
+			if err = setPassword(ctx, u.ID, u.Email, s); err != nil {
+				return err
+			}
+
+			log.Warn(fmt.Sprintf("User %#v created", u))
+
+			//assign the user a BYPASS role
+			for _, r := range internalAuth.BypassRoles() {
+				m := &types.RoleMember{UserID: u.ID, RoleID: r.ID}
+				if err = store.CreateRoleMember(ctx, s, m); err != nil {
+					return err
+				}
+			}
+		}
+	}
+
+	return
+}
+
+// newUser creates and fills a new user depending on the number of arguments provided
+func newUser(user string) *types.User {
+	u := &types.User{
+		ID:        nextID(),
+		CreatedAt: *now(),
+	}
+
+	if strings.Contains(user, ",") {
+		usr := strings.Split(user, ",")
+
+		if len(usr) == 3 {
+			u.Username = usr[0]
+			u.Name = usr[1]
+			u.Email = usr[2]
+			return u
+		}
+
+		u.Username = usr[0]
+		u.Email = usr[1]
```

Comment:
Bold assumption that we can expect 2 elements; is that a requirement? If so, enforce it